packages/manager/modules/dbaas-logs/src/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,3 @@
+{
+  "world": "World!"
```

Comment:
Rebase might be needed as these are the default scaffold translations

Reply:
hum, i remove it ^^